java/runtime/src/main/java/io/ray/runtime/task/NativeTaskSubmitter.java

```diff
@@ -90,14 +89,23 @@ public BaseActorHandle getActor(ActorId actorId) {
   }
 
   @Override
-  public PlacementGroup createPlacementGroup(
-      String name, List<Map<String, Double>> bundles, PlacementStrategy strategy) {
-    byte[] bytes = nativeCreatePlacementGroup(name, bundles, strategy.value());
+  public PlacementGroup createPlacementGroup(PlacementGroupCreationOptions creationOptions) {
+    if (StringUtils.isNotBlank(creationOptions.name)) {
+      PlacementGroup placementGroup =
+          creationOptions.global
+              ? Ray.getGlobalPlacementGroup(creationOptions.name)
+              : Ray.getPlacementGroup(creationOptions.name);
+
+      Preconditions.checkArgument(
+          placementGroup == null,
+          String.format("Placement group with name %s exists!", creationOptions.name));
```

Comment:
Shouldn't GCS check this rather than driver or worker?

Reply:
Currently, the placement group registration is async, the core worker will return `Status::OK` even If the actual registration is failed. but that's a good point because we already have a plan to make the registration sync.